src/main/scala/org/constellation/primitives/Transaction.scala

```diff
@@ -75,6 +75,10 @@ case class Transaction(
   val ordinal: Long = lastTxRef.ordinal + 1L
 }
 
+object Transaction {
+  def oFeeToValue(fee: Option[Long]): Long = fee.map(v => if (v < 0L) 0L else v).getOrElse(0L)
+}
+
```

Comment:
Maybe in the future we could have a Fee class with a smart constructor, something like:
`
  class Fee private(val value: Long) extends AnyVal

  object Fee {
    def apply(value: Long): Fee = if (value < 0L) new Fee(0L) else new Fee(value)
  }
`